app/components/quiz/Quiz.js

```diff
@@ -1,70 +1,66 @@
-import React from 'react';
+import { newQuestions } from 'actions/actions';
+import classNames from 'classnames';
 import QuizQuestion from 'components/quiz/QuizQuestion.js';
-import 'Quiz.scss';
+import React from 'react';
 import ReactCSSTransitionsGroup from 'react-addons-css-transition-group';
+import { connect } from 'react-redux';
+
+import 'Quiz.scss';
 import 'fadeIn.scss';
-import classNames from 'classnames';
 
-export default React.createClass({
+const tmpquestions = [
```

Comment:
no, they're in the store. `tmpquestions` is only used once, on line 99, where it's sent through an action to the store / reducer. since there's no backend to hit for now to get questions, i'm passing this hardcoded stuff in